server/lib/routes/post-metrics-errors.js

```diff
@@ -91,31 +91,21 @@ const QUERY_SCHEMA = {
 /**
  * Attaches extra tags to sentry data
  *
- * @param {String} data - stringified Sentry data object
+ * @param {String} sentryData - stringified Sentry data object
  * @returns {String} data - stringified Sentry data object with extra tags
```

Comment:
`stringified` no longer applies here either it seems. And an `Object` is returned.